web/modules/custom/rdf_entity/modules/rdf_file/rdf_file.services.yml

```diff
@@ -0,0 +1,3 @@
+services:
+  rdf_file.handler:
```

Comment:
I'd prefer to keep this as a service to be able to swap it out:
- It has a contained purpose, that is called from a few places
- The URL is hard coded, so swapping it out might make sense.